UI/DevScreens.cpp

```diff
@@ -423,6 +423,18 @@ const char *GetCompilerABI() {
 	return "x86";
 #elif PPSSPP_ARCH(AMD64)
 	return "x86-64";
+#elif PPSSPP_ARCH(RISCV64)
+    //https://github.com/riscv/riscv-toolchain-conventions#cc-preprocessor-definitions
+    //https://github.com/riscv/riscv-c-api-doc/blob/master/riscv-c-api.md#abi-related-preprocessor-definitions
+    #if defined(__riscv_float_abi_single)
```

Comment:
GetCompilerABI is possibly misnamed, but what we're really looking for isn't the floating point model alone but really the architecture + ABI.

This might be rv64i.  We can keep on the float/long ABI, but the purpose of this is to expose i.e. aemebai vs armeabi-v7a.

-[Unknown]

Reply:
I guess we have a few different options here;
- rv64imafdc 
- rv64gc
- riscv64
- rv64

For more information there's this link here.
https://wiki.debian.org/RISC-V#Hardware_baseline_and_ABI_choice

For simplicity I'd suggest just using rv64gc or RV64GC.
When there is different builds of PPSSPP that target different ISA extensions, we can re-evaluate then.

If you wish to still mention the ABI, that would be yours or @hrydgard's call.

Reply:
Initially you'll probably just want to require floating point support, and not much more (there are embedded RISC-V without it, I think). Not sure what the string for that is though..

Reply:
I think this is just a display to show the user what the build was compiled for.

-[Unknown]

Reply:
Hah oh yeah, didn't look at the context.